### ISSUE TITLE
osgoccluder example bug

### ISSUE THREAD

**Issue by mp3butcher** [openscenegraph/OpenSceneGraph#814]
launching osgoccluder example with current 3.6 branch gives strange results..i doubt that's what expected...

**Comment by mp3butcher** [comment]
https://streamable.com/rv8l5

**Comment by openscenegraph** [comment]
Looking at the video I think this is probably correct.  The osgocculder
example deliberately uses awkward cases that can't cull to test things
out.  With convex planar occluders you have to be able to cull the whole
subgraphs bouding sphere within the occluder, as sooner as one part of the
bounding sphere is outside the occluder it stops occluding and you can then
see the model within.  There is also a whole in one of the box sides, again
as test.

**Comment by mp3butcher** [comment]
i removed the hole to be sure and x10 occluder size but bug it persists not to be culled at occluders junction...well it exhibits that occluder should be oversized in order to work properly...so can be usefull in fact

**Comment by openscenegraph** [comment]
On Wed, 14 Aug 2019 at 17:57, Julien Valentin <notifications@github.com>
wrote:

> i removed the hole to be sure and x10 occluder size but bug it persists
> not to be culled at occluders junction...well it exhibits that occluder
> should be oversized in order to work properly...so can be usefull in fact
>
Convex planar occluders are fickle tools, they only work well in very
specific types of environments where occluders can be very large compared
to the objects they are attempting to cull.

They are a feature in the OSG that we added many years ago but on
implementation we found that they aren't that generally useful.

Robert.
